investpy/__init__.py

```diff
@@ -2011,15 +2173,18 @@ def get_etf_recent_data(etf, country, as_json=False, order='ascending', debug=Fa
                 info.append(nested_.text_content())
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.